pkg/auth/dependency/oauth/handler/handler_authz.go

```diff
@@ -87,7 +92,7 @@ func (h *AuthorizationHandler) doHandle(
 		SessionID:       session.SessionID(),
 
 		CreatedAt: h.Time.NowUTC(),
-		ExpireAt:  h.Time.NowUTC(),
+		ExpireAt:  h.Time.NowUTC().Add(1 * gotime.Minute),
```

Comment:
1 min seems too short. Maybe 3 or 5 minutes?

Reply:
I'll use 5 minutes for now.